src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -43,15 +43,15 @@ class TranslationDebugCommand extends ContainerAwareCommand
     protected static $defaultName = 'debug:translation';
 
     private $translator;
-    private $loader;
+    private $reader;
     private $extractor;
 
     /**
-     * @param TranslatorInterface $translator
-     * @param TranslationLoader   $loader
-     * @param ExtractorInterface  $extractor
+     * @param TranslatorInterface        $translator
+     * @param TranslationReaderInterface $reader
+     * @param ExtractorInterface         $extractor
      */
-    public function __construct($translator = null, TranslationLoader $loader = null, ExtractorInterface $extractor = null)
+    public function __construct($translator = null, TranslationReaderInterface $reader = null, ExtractorInterface $extractor = null)
```

Comment:
isn't this a BC break? BC layer needed instead?

Reply:
hmm the class of the old typehint now implements this interface and is part of FrameworkBundle (which has a conflict for `symfony/translation:<3.4` anyway) so this can't break, right?

Reply:
Ok then :)